.github/workflows/build_wheels.yml

```diff
@@ -0,0 +1,141 @@
+name: Build python wheels
+
+on:
+  push:
+    branches:
+      - master
+      - release-*
+    tags:
+      - v*
+
+jobs:
+
+  build_source:
+    runs-on: ubuntu-18.04
```

Comment:
I am not familiar with github ci. This is a question for my learning. Does each build step (job?) work in a different os environment?

Reply:
> You can specify the runner type for each job in a workflow. Each job in a workflow executes in a fresh instance of the virtual machine. All steps in the job execute in the same instance of the virtual machine, allowing the actions in that job to share information using the filesystem.

source: https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners